Production/crab/config/2018/DY.txt

```diff
@@ -1,11 +1,11 @@
 sampleType=MC_18 applyTriggerMatch=True saveGenTopInfo=False saveGenBosonInfo=True saveGenJetInfo=True
 
-DYJetsToLL_M-50_nlo	                /DYJetsToLL_M-50_TuneCP5_13TeV-madgraphMLM-pythia8/RunIIAutumn18MiniAOD-102X_upgrade2018_realistic_v15-v1/MINIAODSIM
+DYJetsToLL_M-50_nlo	                /DYJetsToLL_M-50_TuneCP5_13TeV-amcatnloFXFX-pythia8/RunIIAutumn18MiniAOD-102X_upgrade2018_realistic_v15-v1/MINIAODSIM
```

Comment:
@kandrosov, DYJetsToLL_M-50_TuneCP5_13TeV-amcatnloFXFX-pythia8 is the nlo sample, right?

Reply:
right, so, in the list there should be 3 inclusive samples: 2 NLO and one LO
`dataset dataset=/DYJetsToLL_M-50_TuneCP5*/RunIIAutumn18MiniAOD-102X_upgrade2018_realistic_v15*/MINIAODSIM`